engines/glk/zcode/detection_tables.h

```diff
@@ -2917,7 +2917,6 @@ const PlainGameDescriptor ZCODE_GAME_LIST[] = {
 	{ "alienabductescape",  "Alien Abduction Escape" },
 	{ "alloneneedscarrot",  "All One Needs Is a Carrot" },
 	{ "almostadream",       "Almost a Dream" },
-	{ "amnesiazcode",       "Amnesia" },
```

Comment:
Hm, in general we don't remove entries evaluating "quality", because it's debatable. Any particular reason why these games should be removed? "The Island" is one of the most played games on the playfic website

Reply:
Indeed, because we also want to preserve and catalogue released games

Reply:
@tag2015 @sev- The point is quite clear: give me some time to fix this "issue"!

Reply:
@tag2015 @sev- ,

https://github.com/scummvm/scummvm/pull/5388/commits/150a995b2c4f070c032fcadb5067b6ed08c3869b
Objection received...

https://github.com/scummvm/scummvm/pull/5388/commits/97eeabd01808352c3a359ba09849b547ba76e100
Meanwhile I've _fully_ checked [**# - A**] entries, [B - Z] wille be done in 2024!

Reply:
Thanks! Looks good to me